rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -417,7 +422,7 @@ def calculate_spend_cost_basis(
             used_acquisitions=used_acquisitions,
             settings=settings,
             timestamp_to_date=timestamp_to_date,
-            average_cost_basis=self.current_average_cost_basis,
+            average_cost_basis=self.current_average_asset_value,
```

Comment:
This looks wrong. Why are you assigning the current average value (which is not the cost basis) to an argument called `average_cost_basis` ?